### PR TITLE
gss.master.accounts

### PR DESCRIPTION
**Pull request by ArtificialOwl**
process to rename `gss.master.admin` to `gss.master.accounts` in config

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: lib/Master.php [hunk 1]
```diff
@@ -143,10 +143,14 @@ public function handleLoginRequest(
 
 		$this->logger->debug('handleLoginRequest: uid is: ' . $uid);
 
-		// let the admin of the master node login, everyone else will redirected to a client
-		$masterAdmins = $this->config->getSystemValue('gss.master.admin', []);
-		if (is_array($masterAdmins) && in_array($uid, $masterAdmins, true)) {
-			$this->logger->debug('handleLoginRequest: this user is a local admin so ignore');
+		// let local account login, everyone else will redirected to a client
```
Comment (artonge) [inline lib/Master.php:146]:
```suggestion
// let local account and admin of the master node login, everyone else will redirected to a client
```

**Review by artonge — COMMENTED** [review]
Seems like this is not a rename but an extension of the feature, no?

File: lib/Master.php (hunk 1 above)
Comment (ArtificialOwl) [reply lib/Master.php:146]: admin are accounts :)

Comment (ArtificialOwl) [reply lib/Master.php:146]: it is more that the feature is currently limited to admin like the configuration parameter might let us think it is

**Pull request merged**